src/components/SecondaryPanes/index.js

```diff
@@ -12,7 +12,7 @@ const { getPause, getBreakpoints,
 const { prefs } = require("../../utils/prefs");
 
 const actions = require("../../actions");
-const WhyPaused = React.createFactory(require("./WhyPaused"));
+const WhyPaused = React.createFactory(require("./WhyPaused").default);
```

Comment:
The `default` here is necessary because we're using the es6 class in a non-es6 like world.  The switch to `import` and JSX would allow us to remove that.  There are also babel plugins to remove this as well but its only temporary.